crates/router/src/compatibility/stripe/errors.rs

```diff
@@ -199,6 +199,8 @@ pub enum StripeErrorCode {
     FileNotFound,
     #[error(error_type = StripeErrorType::HyperswitchError, code = "", message = "File not available")]
     FileNotAvailable,
+    #[error(error_type = StripeErrorType::InvalidRequestError, code = "IR_23", message = "Not Supported because provider is not Router")]
```

Comment:
I don't think we should be sending our error codes via the Stripe compatibility layer.

@Abhicodes-crypto Please confirm.

Reply:
I believe the ideal solution for such cases should be non-centralized API errors which we had taken up earlier...

from a business perspective we should have valid stripe codes for all errors thrown by payments, refunds & customers API

But since we use the same enum for multiple other API endpoints... we can't guarantee that every payment/refund/customer API has a corresponding stripe error code.
At the same time non-stripe compatible API's also need to be mapped to a stripe compatible error (although it's never gonna be used)

Reply:
@SanchithHegde I've removed the error code from the Stripe error stack.